internal/command/state_mv.go

```diff
@@ -385,12 +386,28 @@ func (c *StateMvCommand) Run(args []string) int {
 		return 0 // This is as far as we go in dry-run mode
 	}
 
+	path, err := os.Getwd()
+	if err != nil {
+		c.Ui.Warn(fmt.Sprintf(failedToLoadSchemasMessage, err))
+	}
+
+	config, diags := c.loadConfig(path)
+	if diags.HasErrors() {
+		c.Ui.Warn(fmt.Sprintf(failedToLoadSchemasMessage, err))
+	}
+
+	schemas, diags := getSchemas(&c.Meta, stateTo, config)
+	if diags.HasErrors() {
+		c.Ui.Warn(fmt.Sprintf(failedToLoadSchemasMessage, err))
```

Comment:
We might want to adjust the control flow here; `loadConfig(path)` shouldn't be called if `os.Getwd` errors, and we don't want to print the same warning three times. Same for other instances of this pattern.